### DISCUSSION TITLE
Moving Room Icons

### DISCUSSION THREAD

**Discussion by luk6173** [PiotrMachowski/lovelace-xiaomi-vacuum-map-card#382] [Q&A] [answered]
Hi,

how I can move these room icons to other positions?
![grafik](https://user-images.githubusercontent.com/24668586/163417153-3f882e92-e8fb-4849-a335-82fb1ced2192.png)

Thank you!

**Comment by PiotrMachowski** [comment, marked as answer]
Uhhhhhhh, you just have to change coordinates that you have configured for each icon...

**Comment by luk6173** [comment]
sorry could have come up with it myself!

**Comment by luk6173** [comment]
Other question: How to change the color of this Text?
![grafik](https://user-images.githubusercontent.com/24668586/163430912-54e9760d-02dd-4905-9451-464554721572.png)


Thank you and Regards!

**Reply by PiotrMachowski** [reply]
https://github.com/PiotrMachowski/lovelace-xiaomi-vacuum-map-card/discussions/320
